doc/user_guide/user_guide.rst

```diff
@@ -23,13 +23,13 @@ Needed Repository structure
 ---------------------------
 
 This Project uses Sphinx for the generation of the documentation. Therefore, all files you want to use as a
-source for your documenation have to be compatible with Spinx. You can find Shpinx's
+source for your documentation have to be compatible with Spinx. You can find Sphinx's
 documentation `here <https://www.sphinx-doc.org/en/master/>`_.
 
 In general:
 
 You will need a base directory for your documentation source files. We use "repository_root/doc",
-but you can set it to another folder.
+but you can set it to another folder (Please set the source_dir parameter accordingly. It defaults to /doc/).
```

Comment:
is '/doc/' folder in root directory? or should it be in the project root like './doc/'?

Reply:
Correct for now, see answer below.